spec/factories/item.rb

```diff
@@ -0,0 +1,15 @@
+FactoryGirl.define do
+  factory :item do
+    title
+    description
+    price        5
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.